assets/styles/desktop-styles.css

```diff
@@ -0,0 +1,430 @@
+    font-weight: bold;
+    color: var(--White);
+  }
+
+  .section__hr {
+    width: 50px;
+    margin: auto;
+    margin-top: 10px;
+    border-color: var(--Orange);
+    margin-bottom: 3%;
+  }
+
+  #staff {
+    margin: 7% 0 8%;
+  }
+
+  #staff .flex-container {
+    width: 70%;
+    margin: auto;
+    display: flex;
+    gap: 60px 90px;
+    align-items: flex-start;
+    justify-content: center;
+    flex-wrap: wrap;
+  }
+
+  .staff__head {
+    flex-basis: 100%;
+  }
+
+  #more {
+    display: none;
+  }
+
+  .staff {
+    flex: 1 1 350px;
+  }
+
+  .staff__img img {
+    width: 150px;
+    height: 150px;
+    margin: 12px 20px 0 12px;
+  }
+
+  .staff__description p {
+    font-family: var(--fontFamily-lato);
+    font-weight: normal;
+    font-size: 1.1rem;
+    line-height: 1.5;
+  }
+
+  .toggle {
+    display: flex;
+  }
+
+  #partners-main {
+    display: flex;
+  }
+
+  #partners-main .flex-container {
+    padding: 3% 0 3%;
+    width: 70%;
+    margin: auto;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  #partners {
+    display: none;
+  }
+
+  #partners-main h2 {
+    color: rgb(163, 161, 161);
+    font-size: 1.8rem;
+  }
+
+  #partners-main .section__hr {
+    margin-bottom: 10px;
+  }
+
+  .partner__flex {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 20px;
+  }
+
+  .partner__flex a {
+    color: rgb(163, 161, 161);
+    font-size: 2.3rem;
+    margin-top: 20px;
+  }
+
+  .google,
+  .slack,
+  .airbnb,
+  .soundcloud,
+  .reddit {
+    width: auto;
+    flex: 1 1 80px;
+  }
+
+  #about-hero .flex-container {
+    padding: 10% 0 10%;
+  }
+
+  #about-hero h2 {
+    display: none;
+  }
+
+  #about-hero p {
+    color: var(--Grey);
+    font-size: 1.1rem;
+  }
+
+  #about-logo {
+    margin: 5% 0 0;
+    padding-bottom: 3%;
+  }
+
+  #about-logo h2 {
+    color: var(--Black);
+  }
+
+  #about-logo p {
+    color: var(--Grey);
+    font-weight: normal;
+    font-size: 1.1rem;
+    line-height: 1.5;
+  }
+
+  #about-logo .flex-container {
+    width: 70%;
+  }
+
+  .about-logo__show {
+    margin-top: 3%;
+  }
+
+  .about-logo__show img {
+    width: 50%;
+  }
+
+  #about-past {
+    margin: 3% 0 1%;
+  }
+
+  #about-past .flex-container {
+    width: 70%;
+    margin: auto;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  #about-past h2 {
+    color: var(--Black);
+    font-family: var(--fontFamily-lato);
+  }
+
+  #about-past p {
+    color: var(--Grey);
+    font-weight: normal;
+    font-size: 1.1rem;
+    line-height: 1.5;
+    text-align: center;
+    font-family: var(--fontFamily-lato);
+  }
+
+  .about-past__grid {
+    margin: 5% 0 5% 0;
+    width: 100%;
+    align-items: center;
+    gap: 3%;
+    display: grid;
+    grid-template: 1fr /1fr 1fr;
+  }
+
+  .card1 {
+    min-height: 300px;
+  }
+
+  .card2 {
+    min-height: 300px;
+  }
+
+  #footer {
+    background: rgb(252, 243, 243);
```

Comment:
- [ ] [REQUIRED] As mentioned by the previous reviewer, It will be better to consider following the design by using the required background-color on the footer. section of you about page, This is to make sure that you follow the design guideline and replicate the design to look almost identical to the design guidelines.

In your about page, Your background color for the footer is rgb(252, 243, 243) but the original background color is #272A31

Please fix that and submit for a review.